sass/vapor/_Dropdown.scss

```diff
@@ -10,23 +10,16 @@
   @include flex-basis(115px);
   background: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHByZXNlcnZlQXNwZWN0UmF0aW89InhNaW5ZTWluIiB2aWV3Qm94PSIwIDAgNC45NSAxMCI+PGRlZnM+PHN0eWxlPi5jbHMtMXtmaWxsOiNmZmY7fS5jbHMtMntmaWxsOiM0NDQ7fTwvc3R5bGU+PC9kZWZzPjx0aXRsZT5hcnJvd3M8L3RpdGxlPjxyZWN0IGNsYXNzPSJjbHMtMSIgd2lkdGg9IjQuOTUiIGhlaWdodD0iMTAiLz48cG9seWdvbiBjbGFzcz0iY2xzLTIiIHBvaW50cz0iMS40MSA0LjY3IDIuNDggMy4xOCAzLjU0IDQuNjcgMS40MSA0LjY3Ii8+PHBvbHlnb24gY2xhc3M9ImNscy0yIiBwb2ludHM9IjMuNTQgNS4zMyAyLjQ4IDYuODIgMS40MSA1LjMzIDMuNTQgNS4zMyIvPjwvc3ZnPg==)
     no-repeat;
-  background-position-x: 95%;
-  background-position-y: 50%;
+  background-position: right 5px top 50%;
+  background-size: 16px 32px;
   vertical-align: middle;
   height: 30px;
   min-width: 150px;
-  padding: 0 20px 0 0;
+  padding: 0 25px 0 0;
```

Comment:
This padding better accommodates the new slightly larger size of the icon:
![image](https://user-images.githubusercontent.com/54454747/103919887-981e0c80-50de-11eb-8a9e-fef0f2eeb0c6.png)